.github/ISSUE_TEMPLATE/issue_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Issue report
+description: Create a report to help us improve
+labels: ["issue"]
```

Comment:
Also lets add a `feature.md` as well. example: https://github.com/facebook/jest/blob/main/.github/ISSUE_TEMPLATE/feature.yml

Reply:
i have added feature template could you please check once
https://github.com/razorpay/razorpay-node/blob/issue-template-v1/.github/ISSUE_TEMPLATE/feature.yml

Reply:
Looks good!